charts/sscs-tribunals-api/values.preview.template.yaml

```diff
@@ -65,10 +65,11 @@ java:
     HOURS_END_TIME: 23
     RUN_DB_MIGRATION_ON_STARTUP: true
     TEST_RECIPIENTS_POSTCODE: "*"
-    INFECTED_BLOOD_COMPENSATION_ENABLED: true
+    INFECTED_BLOOD_COMPENSATION_ENABLED: false
     SUBMIT_APPEAL_SERVICE_V2_ENABLED: true
     PROCESS_EVENT_MESSAGE_V2_ENABLED: true
     UPDATE_CASE_ONLY_HEARING_V2_ENABLED: true
+    TRIGGER_EVENT_V2_ENABLED: true
     # START - SSCS-IB RELEASE VARIABLES
```

Comment:
why did you need to enable this?

Reply:
This was added to enable AppealReceivedHandler to take V2 path - triggerCaseEventV2

Reply:
As discussed enabled it on values.yaml